ext/intl/uchar/uchar.c

```diff
@@ -173,8 +173,8 @@ typedef struct _enumCharType_data {
 	zend_fcall_info_cache fci_cache;
 } enumCharType_data;
 static UBool enumCharType_callback(enumCharType_data *context,
-                                   UChar32 start, UChar32 limit,
-                                   UCharCategory type) {
+								   UChar32 start, UChar32 limit,
+								   UCharCategory type) {
```

Comment:
Not really related to your change, but please get rid of the alignment here, it doesn't work with tabs.

```
foobar(
    foo, bar)
// instead of
foobar(foo,
       bar)
```